src/apps/SDIOutApp/SDIOutApp.js

```diff
@@ -47,7 +47,7 @@ class SDIOutApp extends Component {
                 })
                 .catch(err => {
                     console.error("[SDIOut] error fetching quad state", err);
-                    captureException(err, {err: `[SDIOut] error fetching quad state: ${err}`});
+                    captureException(err, {source: "SDIOut"});
```

Comment:
I'm missing here a way to add the 49th line context to the capture... or we can assume breadcrumbs will be added.... WDYT?